engine/class_modules/sc_death_knight.cpp

```diff
@@ -4670,6 +4674,11 @@ struct empower_rune_weapon_t : public death_knight_spell_t
 
     cooldown -> duration *= 1.0 + p -> vision_of_perfection_minor_cdr;
     cooldown -> duration += p -> spec.empower_rune_weapon_2->effectN( 1 ).time_value();
+    if ( p -> conduits.accelerated_cold->ok() )
```

Comment:
"recharges X% faster" isn't the same as "reduces the cooldown by X%" (ie. recharing 100% faster is equal to reducing cooldown by only 50%).

For this, you'll have to override action_t::recharge_multiplier() and apply the modifier there

Reply:
I moved it into the override, as I do think it makes it more clear.

This one does look to have multiple issues.

1) The tooltip is wrong.  When we look at the spell data the two effects are referrenced in the wrong order in the tooltip.
2)  The value for affect 2 is listed as being negative, as well as being "Modify Recharge Time% (Category) (454)".
3)  I was unable to find anything else that is 454 in the DB.

4)  sc_buff.cpp:1280 has the section for 454 commented, out, with the following code:
```
        // TODO: Should buffs support a base_recharge_multiplier?
        //base_recharge_multiplier *= 1 + effect.percent();
        //if ( base_recharge_multiplier <= 0 )
        //  set_cooldown( timespan_t::zero() );
        //sim->print_debug( "{} cooldown recharge multiplier modified by {}%", *this, effect.base_value() );
```

Though we can find it in 
sc_action.cpp:4917 implemented as:
```
base_recharge_multiplier *= 1 + effect.percent();
sim->print_debug( "{} cooldown recharge multiplier modified by {}%", *this, effect.base_value() );
```

So the newest one should match implementation to what the apply_affecting_aura is doing.

I'd fully expect to see this particular spelldata to be modified and fixed at some point, so I will likely need to revist this.

```
SimulationCraft 901-01 for World of Warcraft 9.0.2.35854 shadowlands-BETA (hotfix 2020-09-14/35854, git build frost_conduits 600e34c46b)

Name             : Accelerated Cold (id=337822) [Spell Family (15), Passive, Hidden]
School           : Physical
Spell Type       : None
Conduit Id       : 79 (values=5.0, 5.5, 6.0, 6.75, 7.5, 8.25, 9.0, 9.75, 10.5, 11.25, 12.0, 12.75, 13.5, 14.25, 15.0)
Attributes       : ......xx ........ ........ ........   ........ ........ ........ ........
                 : ........ ........ ........ ........   ........ ........ ........ ........
                 : ........ ........ ........ ........   ........ ........ ........ ........
                 : ........ ........ ........ ........   ........ ........ ........ ........
                 : ........ ........ ........ ........   ........ ........ ........ ........
                 : ........ ........ ........ ........   ........ ........ ........ ........
                 : ........ ........ ........ ........   ........ ........ ........ ........
                 : ...x.... ........ ........ ........
                 : Passive (6), Hidden (7)
Effects          :
#1 (id=836041)   : Apply Aura (6) | Add Flat Modifier (107): Spell Effect 3 (23)
                   Base Value: 5 | Scaled Value: 5 | PvP Coefficient: 1.00000 | Misc Value: 23 | Target: Self (1)
                   Affected Spells: Empower Rune Weapon (47568), Hungering Rune Weapon (207127)
                   Family Flags: 14
#2 (id=836047)   : Apply Aura (6) | Modify Recharge Time% (Category) (454)
                   Base Value: -10 | Scaled Value: -10 | PvP Coefficient: 1.00000 | Misc Value: 1614 | Target: Self (1)
                   Affected Spells (Category): Empower Rune Weapon (47568), Hungering Rune Weapon (207127)
Description      : Empower Rune Weapon recharges $s1% faster and the Haste amount is increased by |cFFFFFFFF${$s2}.1%|r.
```